README.md

```diff
@@ -69,7 +70,8 @@ Navigate back to the root directory and run the Postgres docker container:
 
 ```
 $ cd ..
-$ docker run -d -p 5432:5432 --name peer-sql peer-sql
+$ docker network create peer-network
```

Comment:
Hmm.. is the `peer-network` line necessary?  I've just been running the containers over bridge.  Not opposed to including, it just curious as to the reasoning.

Also, there are minikube deployments in `/infrastructure/kubernetes/development/` and instructions for running them in `/documentation/running-development.md`.  The intention is that the minikube environment would serve as the local "development" environment, in addition to to the "hot-dev" environment, rather than running docker containers raw as it were.

The "development" config file in `/config/` is written with this in mind.  

That said, there is definitely value in being able to run the containers raw.

Reply:
Sorry that I missed `/documentation/running-development.md`! Using Kubernetes would of course also have worked.

With pure Docker, without specifying a user-defined network/bridge, the container can reach each other only through their IP addresses but not by the container names ([docs](https://docs.docker.com/network/bridge/#differences-between-user-defined-bridges-and-the-default-bridge)) and the default development config sets the database host name to `peer-review-database-service`. This reminds me that I would also like to propose changing the database container name in the README to `peer-review-database-service` so that it is easy to get the project running without changing any configs (already force-pushed).

But of course, I'm happy to adapt the PR as you wish!

Reply:
Oh neat!  This is the thing I love/hate about DevOps.  I ran a whole DevOps team for four years - never noticed that functionality of docker (to be fair, we were only half involved in the local environments, but still!)  Thanks for teaching me something :)

One note, when I tried the `docker run` on line 74, I got an error, because I think `-net` needs to be `--net`.  

Once you fix that, these changes look good! 

...but... uh...  you probably already noticed that I broke the ability to run on local (with out access keys) when I implemented Digital Ocean Spaces.  I was pushing hard to get things working, and taking on techdebt to do it, so I'll need to go back and fix that to enable local file storage again as an option.  

Alternately, you can create a digital ocean space and add that to the config file.  I didn't document the process of creating a bucket, only the process of adding in the access keys.  There is terraform to create it, but some parameters will need to be changed.  And it costs money - so it would probably be better to get local file storage working again. 

 It shouldn't be too hard.  Just need to add in some code forks in a few places - oh well, and figure out how to make it place nice with minikube through mounting volumes.  That's going to be a headache.  But getting it working on hot-dev should be pretty straight forward, since it was working before I moved to Spaces.

If you wanted to work on that and submit a PR, I'd be happy to look at it! Otherwise I can try to get to it (and updating this documentation) later this week :)

Reply:
Thanks for noticing the typo, I just fixed it.

Regarding the storage: I don't think that I will be able to work on that very soon - sorry!

Reply:
No worries!  I'll try to get to it this week :)  Thanks for these fixes!